src/views/GroupList/GroupList.tsx

```diff
@@ -66,7 +69,9 @@ export class GroupList extends React.PureComponent {
                             name="game-history"
                             source={`groups/`}
                             orderBy={["-member_count"]}
-                            filter={{ "name__icontains": "" }}
+                            filter={{
+                                ...(this.state.name_contains_filter !== "" && {'name__icontains': this.state.name_contains_filter})
```

Comment:
This syntax  (`...(some_obj)` is cool I'd never seen that.

One note- it doesn't hurt to be explicit with the check, but `PaginatedTable *does* do some handling for empty strings in case you were wondering how the above line was possible before:

https://github.com/online-go/online-go.com/blob/52772889d53c97f038cea90020a6c9698bdee214/src/components/PaginatedTable/PaginatedTable.tsx#L132-L138